instrumentation/kafka/kafka-clients/kafka-clients-common-0.11/library/src/main/java/io/opentelemetry/instrumentation/kafkaclients/common/v0_11/internal/KafkaConsumerAttributesExtractor.java

```diff
@@ -49,6 +51,11 @@ public void onStart(
     if (consumerGroup != null) {
       attributes.put(MESSAGING_KAFKA_CONSUMER_GROUP, consumerGroup);
     }
+
+    String bootstrapServers = request.getBootstrapServers();
+    if (bootstrapServers != null) {
+      attributes.put(MESSAGING_KAFKA_BOOTSTRAP_SERVERS, bootstrapServers);
+    }
```

Comment:
By default we emit only attributes that are in spec. Either get it added to spec or move to `KafkaConsumerExperimentalAttributesExtractor`